index.html

```diff
@@ -0,0 +1,292 @@
+    <main>
+      <section class="headline" id="homePage">
+        <div class="text">
+          <h3 class="small-title">Welcome to the amazing</h3>
+          <h1 class="big-title">Villa Iluminada Atlixco, Puebla 2021</h1>
+          <p class="paragraph">
+            This tradition started in 2011 and ever since, it has been one of
+            the most awaited events in the locality. People from other states of
+            the country come each December to admire the beauty and the magic of
+            the lights.
+          </p>
+          <p class="last-info">2021.11.27(SAT) ~ 2022.01.06(THU)</p>
+          <p class="last-info">@villailuminada Atlixco,Pue.</p>
+        </div>
+      </section>
+      <section class="program">
+        <h3 class="main">Main Program</h3>
+        <div class="underscore"></div>
+        <section class="row">
+          <div class="atractions">
+            <img src="Images/parade.svg" alt="horse-icon" height="100" />
+            <h4 class="red-title">Parade</h4>
+            <p class="program-paragraph">
+              The event counts with an incredible parade every single night
+              where the magic feels in every corner of this town.
+            </p>
+          </div>
+          <div class="atractions">
+            <img src="Images/christmas-tree.png" alt="tree-icon" height="100" />
+            <h4 class="red-title">Christmas</h4>
+            <p class="program-paragraph">
+              Christmas is the best season for many people, and in this event is
+              a great opportunity to discover the magic of the Christmas.
+            </p>
+          </div>
+          <div class="atractions">
+            <img src="Images/fast-food.svg" alt="food-icon" height="100" />
+            <h4 class="red-title">Food & Drinks</h4>
+            <p class="program-paragraph">
+              The event counts with many different kind of food. We have
+              different stands that offers to you a new opportunity to discover
+              a new taste.
+            </p>
+          </div>
+          <div class="atractions">
+            <img
+              src="Images/souveniers.svg"
+              alt="souveniers-icon"
+              height="100"
+            />
+            <h4 class="red-title">Souveniers</h4>
+            <p class="program-paragraph">
+              In the whole town you are going to discover many different
+              establishments where you can get souveniers.
+            </p>
+          </div>
+          <div class="atractions">
+            <img src="Images/lights.svg" alt="lights-icon" height="100" />
+            <h4 class="red-title">Lights</h4>
+            <p class="program-paragraph">
+              The principal tematic of this event definitely are the lights.
+              Watching the whole town illuminated is the principal reason that
+              many people come in this season every year.
+            </p>
+          </div>
+        </section>
```

Comment:
- [ ] **The overall spacing within the `Main Program` section is still significantly different than the original. Please fix this.**
 
If you notice the original `Main Program` section above, it has more spacing within the element on top and bottom.  Also, the margin between each element can be slightly increased. 

Hint: Look for a CSS rule that will create more spacing inside the element.